src/cargo/core/compiler/fingerprint.rs

```diff
@@ -812,7 +812,7 @@ impl Fingerprint {
         *self.memoized_hash.lock().unwrap() = None;
     }
 
-    fn hash(&self) -> u64 {
+    fn hash_u64(&self) -> u64 {
```

Comment:
I renamed this because there were two functions with the same name but different signatures (the inherent fn and the trait). This caused a problem when importing the trait, and was also confusing.